apps/www/data/PricingAddOnTable.json

```diff
@@ -15,10 +15,10 @@
           { "key": "cpu", "title": "CPU", "value": "2-core ARM" },
           { "key": "dedicated", "title": "Dedicated", "value": false },
           { "key": "memory", "title": "Memory", "value": "1 GB" },
-          { "key": "maxDiskIO", "title": "Disk IO Bandwidth up to", "value": "2,606 Mbps" },
+          { "key": "maxDiskIO", "title": "Max Disk Mbps", "value": "2,085 Mbps" },
```

Comment:
`Max Disk Mbps` Shouldn't there be something like bandwidth in there? Feel like it's missing a word